src/models/users.model.js

```diff
@@ -0,0 +1,124 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    minlenght: 3,
+    maxlenght: 25,
+    required: true,
+  },
+  lastname: {
+    type: String,
+    minlenght: 3,
+    maxlenght: 25,
+    required: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  email: {
+    type: String,
+    required: true,

```

Comment:
Necesita un regex, literal puedo ingresar alejandra.hola y me lo va a aceptar.

```suggestion
    required: true,
    match: /^.*@.*\..*$/,
```

O, si quieres otro: https://ihateregex.io/

Reply:
Si, de acuerdo, añado uno.